x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateTimeProcessorTests.java

```diff
@@ -31,12 +35,12 @@ protected DateTimeProcessor createTestInstance() {
     }
 
     @Override
-    protected DateTimeProcessor mutateInstance(DateTimeProcessor instance) throws IOException {
+    protected DateTimeProcessor mutateInstance(DateTimeProcessor instance) {
         DateTimeExtractor replaced = randomValueOtherThan(instance.extractor(), () -> randomFrom(DateTimeExtractor.values()));
-        return new DateTimeProcessor(replaced, UTC);
+        return new DateTimeProcessor(replaced, randomValueOtherThan(UTC, ESTestCase::randomZone));
```

Comment:
Why do you use here `random other than UTC`?

Reply:
No reason, I will change.